backend/dal/Services/Concrete/BuildingService.cs

```diff
@@ -223,6 +223,8 @@ public Building Update(Building building)
             var userAgencies = this.User.GetAgenciesAsNullable();
             if (!isAdmin && !userAgencies.Contains(existingBuilding.AgencyId)) throw new NotAuthorizedException("User may not edit buildings outside of their agency.");
 
+            if (!isAdmin && existingBuilding?.ProjectNumbers?.Contains("SPP") == true) throw new NotAuthorizedException("User may not edit buildings that are in a SPP Project.");
```

Comment:
I don't love depending on the not enforced project numbers like this, but at the same time multiple areas on the frontend and the backend are already doing this so...